hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/JsonUtil.java

```diff
@@ -741,4 +741,27 @@ public static Map<String, Object> toJsonMap(FsStatus status) {
     m.put("remaining", status.getRemaining());
     return m;
   }
+
+  public static Map<String, Object> toJsonMap(ErasureCodingPolicyInfo ecPolicyInfo) {
+    if (ecPolicyInfo == null) {
+      return null;
+    }
+    Map<String, Object> m = new HashMap<>();
+    m.put("policy", ecPolicyInfo.getPolicy());
+    m.put("state", ecPolicyInfo.getState());
+    return m;
+  }
+
+  public static String toJsonString(ErasureCodingPolicyInfo[] ecPolicyInfos) {
+    final Map<String, Object> erasureCodingPolicies = new TreeMap<>();
```

Comment:
Why TreeMap?

Reply:
Thank you for your valuable suggestion. Since sorting of keys is not required, it's unnecessary to use `TreeMap`.  I sincerely appreciate it and will promptly implement the required adjustments to the code!